spec/libraries/aws_engine_spec.rb

```diff
@@ -25,6 +28,7 @@ def initialize
           @msg = []
           @msg.push(DummyMessage.new)
         end
+
```

Comment:
__[reek]__
* [UnusedParameters] has unused parameter 'params' [:ref](https://github.com/troessner/reek/blob/master/docs/Unused-Parameters.md)
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)